### PR TITLE
Feat/workflowlist

### PR DESCRIPTION
**Pull request by starkris51**
Includes #85, plus: 
1. workflow list
2. success page

### CODE REVIEW

**Comment by fredrikved** [comment]
You can base a PR on another branch if the changes are based on the commits from the other branch connected to another PR.
Easier to see the changes then, and still keep the discussion about changes.

File: cmd/trigger_ui/list.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                                    <td class="px-5 py-5 border-b border-gray-200 bg-white text-sm">
+                                        <p class="text-gray-900 whitespace-no-wrap">
+                                            {{$element.WorkflowID}}
+                                        </p>
+                                    </td>
+                                    <td class="px-5 py-5 border-b border-gray-200 bg-white text-sm">
+                                        <p class="text-gray-900 whitespace-no-wrap">
+                                            {{$element.Start}}
+                                        </p>
+                                    </td>
```
Comment (fredrikved) [inline cmd/trigger_ui/list.html:69]: Youre also assigning bg-white and text-sm on every child element. This is inherited from the parent, so you can assign common classes to the parent element. <tr> or <tbody> here.

File: cmd/trigger_ui/list.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                                            <span
+                                            class="relative inline-block px-3 py-1 font-semibold text-yellow-800 leading-tight">
+                                            <span aria-hidden
+                                                class="absolute inset-0 bg-yellow-200 opacity-50 rounded-full"></span>
+                                        {{end}}
+                                        <span class="relative">{{$element.Status}}</span>
+                                        </span>
+                                        
+                                    </td> 
+
```
Comment (fredrikved) [inline cmd/trigger_ui/list.html:107]: Is there a missing </tr> tag here?

File: cmd/trigger_ui/list.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                                        class="px-5 py-3 border-b-2 border-gray-200 bg-gray-100 text-left text-xs font-semibold text-gray-600 uppercase tracking-wider">
+                                        Status
+                                    </th>
+                                </tr>
+                            </thead>
+
+                            {{range $index, $element := .WorkflowList}}
+                            
+                            <tbody>
+                                <tr>
```
Comment (fredrikved) [inline cmd/trigger_ui/list.html:48]: You should probably put the range inside the tbody element...

File: cmd/trigger_ui/list.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                                                        {{$element.VxID}}
+                                                    </p>
+                                    </td>
+
+                                    <td class="px-5 py-5 border-b border-gray-200 bg-white text-sm">
+                                        <p class="text-gray-900 whitespace-no-wrap">
+                                            {{$element.Name}}
+                                        </p>
+                                    </td>
+                                    <td class="px-5 py-5 border-b border-gray-200 bg-white text-sm">
```
Comment (fredrikved) [inline cmd/trigger_ui/list.html:60]: `px-5 py-5` can also be written as `p-5`

`border-b` can be replaced by class on tbody `<tbody class="divide-y">`

**Review by fredrikved — CHANGES_REQUESTED** [review]

File: cmd/trigger_ui/list.html [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+                                        Workflow ID
+                                    </th>
+                                    <th
+                                        class="px-5 py-3 border-b-2 border-gray-200 bg-gray-100 text-left text-xs font-semibold text-gray-600 uppercase tracking-wider">
+                                        Start
+                                    </th>
+                                    <th
+                                        class="px-5 py-3 border-b-2 border-gray-200 bg-gray-100 text-left text-xs font-semibold text-gray-600 uppercase tracking-wider">
+                                        Status
+                                    </th>
```
Comment (fredrikved) [inline cmd/trigger_ui/list.html:41]: A lot of duplicate classes here as well. You can declare most of it on the parent div.

File: cmd/trigger_ui/list.html [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+                                </tr>
+                            </thead>
+
+                            
+                            
+                            <tbody class="divide-y">
+                                {{$WorkflowStatuses := .WorkflowStatuses}}
```
Comment (fredrikved) [inline cmd/trigger_ui/list.html:48]: Shouldn't need to declare variable here when you use it later. Just reference `.WorkflowStatuses` directly

**Review by fredrikved — APPROVED** [review]
Still some minor things, but other than that looks okay.

File: cmd/trigger_ui/list.html (hunk 6 above)
Comment (starkris51) [reply cmd/trigger_ui/list.html:48]: the loop inside the loop will only work if the variable is declared outside

**Pull request merged**